notebooks/boston_light_swim/00-fetch_data.ipynb

```diff
@@ -60,9 +60,11 @@
    "source": [
     "%%writefile config.yaml\n",
     "\n",
+    "# If not specified defaults to today +- 3 days.\n",
+    "# Use the date below to reproduce the last Boston Light Swim event forecast.\n",
     "date:\n",
-    "    start: 2016-8-16 00:00:00\n",
-    "    stop: 2016-8-29 00:00:00\n",
+    "    start: null  # 2016-8-16 00:00:00\n",
+    "    stop: null  # 2016-8-29 00:00:00\n",
```

Comment:
@rsignell-usgs this is the main change. The user is free to entry any `start`/`stop` dates. If one of those is undefined the other one will be set to &pm; 3 days. If both are undefined then it will be `today` &pm; 3 days.

Is that OK?